### PR TITLE
Add option for printing rolled-up feature weight avgs

### PR DESCRIPTION
**Pull request by OptimusRhine**
* added parameter-based option for printing rolled-up feature weight averages for given model and test data
  * saves + prints DF with both mean and bayes averages

### CODE REVIEW

File: retain_interpretations.py [hunk 1]
```diff
@@ -210,6 +215,12 @@ def get_predictions(model, data, model_parameters, ARGS):
                                     use_multiprocessing=True, verbose=1, workers=3)
     return preds
 
+def bayesian_average(df):
```
Comment (WhiteAu) [inline retain_interpretations.py:218]: much cleaner. Nice

File: retain_interpretations.py [hunk 2]
```diff
@@ -221,21 +232,42 @@ def main(ARGS):
...
+                # update progress bar
+                pbar.update(1)
+        # averaged across all patients
+        full_avgs = pd.DataFrame(index=dictionary.values())
+        full_avgs['Mean'] = mean_avgs.mean(axis=1)
+        full_avgs['Bayes'] = bayesian_average(mean_avgs)
+        full_avgs.to_pickle('feature_weights.pkl')
+        print(full_avgs.to_string())
+    else:
+        while 1:
```
Comment (WhiteAu) [inline retain_interpretations.py:257]: Does this never break?

**Review by WhiteAu — APPROVED** [review]
looks good. One small question about whether the while 1: loop ever breaks.

File: retain_interpretations.py (hunk 2 above)
Comment (OptimusRhine) [reply retain_interpretations.py:257]: Only when the user breaks it. This is a repeating prompt for user inputted PatientID, so they can look up results for specific patients.

This was all part of the original retain-keras (see changelog); I just put it inside an else statement.

**Review by WhiteAu — APPROVED** [review]
looks great

**Pull request merged**